src/VuData.h

```diff
@@ -182,6 +182,13 @@ class Vu  : public P8PLATFORM::CThread
   std::vector<VuTimer> LoadTimers();
   void TimerUpdates();
   bool GetDeviceInfo();
+	struct {
+		long long total, used;
+		//30 secs btw. refreshes is enough for PVR
+		const int interval = 30;
+		long lastrefresh;
+	} m_driveSpace;
+	void ParseDriveSpace(TiXmlElement* pNode);
```

Comment:
Wrong indentation here

Reply:
sorry, that was Eclipse's formatting and auto format is like a reflex to me. 
I've commited a manual try to the branch